frontend/src/styles/globals.css

```diff
@@ -38,3 +38,6 @@
   @apply !border-r-black
 }
 
+.details-table-first-column-border {
```

Comment:
Why are you declaring a class here?

Reply:
Uff it looks like I have pushed my test branch, not the correct one. I'll take a look.

Reply:
@andresgnlez 

In these first column cells, the bottom border is solid (much like in all cells), but in these particular ones the right border needs to be dashed. I haven't found a way to pull this off the regular tailwind way, hence the custom style. 

Let me know if you know of a way.